app/models/file_download_stat_decorator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# OVERRIDE Hyrax hyrax-v3.5.0 to require Hyrax::Download so the method below doesn't fail
+
+require 'hyrax/download'
```

Comment:
I _think_ you want this to require_dependency here https://apidock.com/rails/ActiveSupport/Dependencies/Loadable/require_dependency